cmd/root.go

```diff
@@ -226,6 +228,17 @@ func parseConfig(cmd *Command, conf *proxy.Config, args []string) error {
 	if conf.CredentialsFile != "" && conf.GcloudAuth {
 		return newBadCommandError("cannot specify --credentials-file and --gcloud-auth flags at the same time")
 	}
+	if userHasSet("host") && conf.Host != "" {
```

Comment:
Since we're setting a default value for `Host`, it will always be a non-empty string so the second part of this boolean statement can be removed.